s3/compare-credentials-jumpbox.sh

```diff
@@ -0,0 +1,16 @@
+# Pass the credential file as an argument
+main() {
+    CREDENTIAL_FILE=$(basename $1)
+    aws s3 cp --sse AES256 s3://concourse-credentials/$CREDENTIAL_FILE test-file-s3.yml
+    diff $1 test-file-s3.yml
+    rm test-file-s3.yml
+    echo "do you want to upload? y/n"
+    read response
+    if [[ $response = "y" ]]
+    then
+      aws-vault exec gov-prd-plat-admin -- aws s3 cp --sse AES256 $1 s3://concourse-credentials/$CREDENTIAL_FILE
```

Comment:
```suggestion
      aws s3 cp --sse AES256 $1 s3://concourse-credentials/$CREDENTIAL_FILE
```

We shouldn't specify the role in the script. You should pass the role in order to call the script:

```
aws-vault exec <profile> -- s3/compare-credentials-jumpbox.sh
```